src/main/java/gregtech/common/tools/ToolJackHammer.java

```diff
@@ -21,12 +21,14 @@
 import net.minecraft.world.World;
 import net.minecraftforge.common.util.FakePlayer;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
 
 public class ToolJackHammer extends ToolDrillLV {
 
+    private static final Set<String> HAMMER_TOOL_CLASSES = new HashSet<String>() {{
+        add("pickaxe"); add("hammer");
+    }};
```

Comment:
Same thing as other comment, perhaps also should be called `JACKHAMMER_[...]` rather than `HAMMER_[...]`

Reply:
It uses "hammer" in canMineBlock, so it should be consistent:
https://github.com/GregTechCE/GregTech/blob/d9e7ed4570b990ed389a6a96589636ced78edd34/src/main/java/gregtech/common/tools/ToolJackHammer.java#L117

Reply:
All these if statements could be changed to something like:
````
return HAMMER_TOOL_CLASSES.contains(tool) ||
````
It will be slightly less efficient, but easier to maintain in future.
i.e. you won't have subtle bugs introduced by inconsistencies between getToolClasses() and canMineBlock()

Reply:
> It uses "hammer" in canMineBlock, so it should be consistent:

I clearly meant the name of the constant, not the value of the strings in the Set.

> All these if statements could be changed to something like:

Not sure what this has to do with this block of code; perhaps create a separate "conversation" linked to the code you're talking about so we have the appropriate context.